crates/wash-lib/src/registry.rs

```diff
@@ -69,6 +71,12 @@ pub enum SupportedArtifacts {
     Wasm(Config, ImageLayer),
 }
 
+/// An enum indicating the type of artifact that was pulled
+pub enum ArtifactType {
```

Comment:
Does it make sense for this to be `OciArtifactType`? Are we going to have any other kind of registry/`pull`able area?

Reply:
I was following similar naming to `SupportedArtifacts`, but since this is in the registry module, I don't think we need the additional context